api/src/main/resources/application.properties

```diff
@@ -0,0 +1,12 @@
+spring.datasource.url=jdbc:postgresql://localhost:5432/minibcr
+spring.datasource.username=houssam.mahlous
+spring.datasource.password=postgres
```

Comment:
We never store the values in application.properties file and commit them to github. They should be empty, but when you work locally you can set them up yourself.

Reply:
I will go ahead and store the values in  a local .env file instead